lms/djangoapps/mobile_api/users/serializers.py

```diff
@@ -89,10 +89,11 @@ class CourseEnrollmentSerializer(serializers.ModelSerializer):
 
     def get_certificate(self, model):
         """Returns the information about the user's certificate in the course."""
-        certificate_info = certificate_status_for_student(model.user, model.course_id)
-        if certificate_info['status'] == CertificateStatuses.downloadable:
+        certificate_info = certificate_downloadable_status(model.user, model.course_id)
+        if certificate_info['is_downloadable']:
+            request = self.context.get('request', None)
             return {
-                "url": certificate_info['download_url'],
+                "url": request.build_absolute_uri(certificate_info['download_url']),
```

Comment:
This will blow up if request is None.  Consider applying `build_absolute_uri` conditionally if request is not None, but allowing basic functionality otherwise.